packages/mask/src/plugins/Collectible/base.ts

```diff
@@ -16,6 +16,7 @@ export const base: Plugin.Shared.Definition = {
         postContent: new Set([
             /opensea.io\/assets\/(0x[\dA-Fa-f]{40})\/(\d+)/,
             /rarible.com\/token\/(0x[\dA-Fa-f]{40}):(\d+)/,
+            /marketplace.treasure.lol\/collection\/(0x[\dA-Fa-f]{40})\/(\d+)/,
```

Comment:
It does match the following links:

+ https://marketplace.treasure.lol/collection/legion-auxiliary/597
+ https://marketplace.treasure.lol/collection/legion-genesis/2635

Reply:
Should be fixed now